hw3/Author.rb

```diff
@@ -0,0 +1,36 @@
+class Author
+    attr_reader :name, :biography, :yearOfBirth
+    
+    def initialize(name, yearOfBirth, biography='')
+        unless validName?(name) 
+            raise 'Not valid name'  
+        end    
+        unless validYearOfBirth?(yearOfBirth)
+            raise "Not valid yearOfBirth"
+        end    
+        @name = name
+        @yearOfBirth = yearOfBirth
+        @biography = biography   
+    end
+
+    def validName?(name)
+        return true if (name.length >= 3 && name.length <= 15)
```

Comment:
- name.length >= 3 && name.length <= 15
- name.length.in?(3..15)
- use constant for (3..15)